src/openforms/js/components/formio_builder/builder.js

```diff
@@ -161,13 +161,19 @@ const getBuilderOptions = () => {
               input: true,
               inputMask: '9999 AA',
               validateOn: 'blur',
+              // The validate config doesn't seem to be used. It's also missing in the form definition BD
+              // See for more information: https://github.com/open-formulieren/open-forms/issues/4719#issuecomment-2429166434
```

Comment:
This sounds like a bug to me - it's definitely unexpected behaviour. Could be something that broken because of the formio-builder transition, I'll take a look.

Reply:
@robinmolen I pushed baef18e9d7990297151eb3cba70efc16ab68878b for this, can you do a double check which set of changes/workarounds are still needed with this fix of mine applied?

Reply:
With your fix, only the translations seem to be needed. I'll remove the config changes from the commit 👍